Talent portal

milstan opened the pull request:
Closes blindnet-io/product-management/issues/993

milstan committed:
init talent portal

milstan committed:
talent portal v1

m4rk055 commented on talent_portal/js/submit.js:
```diff
@@ -0,0 +1,72 @@
+            id: formData.get('email'),
+            schema: 'dsid',
+          },
+          consentId: '28b5bee0-9db8-40ec-840e-64eafbfb9ddd',
```
This is the same consent from the demo app. Shouldn't we have a different one?

milstan replied:
Changed to a new consent

m4rk055 commented on talent_portal/js/submit.js:
```diff
@@ -0,0 +1,72 @@
+    });
+
+    // submit data
+    fetch('https://blindnet-connector-demo-staging.azurewebsites.net/form', {
```
Saving the data should be done only after the consent is successfully registered. Put this code inside of `then` of the previous `fetch` call.

milstan replied:
done

m4rk055 commented on talent_portal/thanx.html:
```diff
@@ -0,0 +1,59 @@
+
+      <div class="row h-75 align-self-center align-items-center justify-content-center">
+        <div class="col-md-6 col-lg-5 text-center">
+          <h1>thanx!</h1>
```
```suggestion
          <h1>thanks!</h1>
```

milstan committed:
Update talent_portal/thanx.html

Co-authored-by: Marko <71560916+m4rk055@users.noreply.github.com>

milstan committed:
fixes requested by @m4rk055

New consent ID created for talent contact form

Form submitted only after registering consent

milstan committed:
Merge branch 'talent-portal' of https://github.com/blindnet-io/dogfood into talent-portal
